app/src/main/java/org/schabi/newpipe/info_list/holder/PlaylistMiniInfoItemHolder.java

```diff
@@ -36,7 +37,8 @@ public void updateFromItem(final InfoItem infoItem, final HistoryRecordManager h
         final PlaylistInfoItem item = (PlaylistInfoItem) infoItem;
 
         itemTitleView.setText(item.getName());
-        itemStreamCountView.setText(String.valueOf(item.getStreamCount()));
+        itemStreamCountView.setText(Localization.localizeStreamCount(itemStreamCountView.getContext(),
```

Comment:
Here it should say just the count like in my PR. So e.g. "100+" instead of "100+ videos".

Reply:
No, to keep consistenct, the string should contain "videos":
"No videos"
"1 video"
"2 videos"
...
"100+ videos"
"∞ videos"

Reply:
Do you really think there's enough space for that there? 
![Screenshot](https://user-images.githubusercontent.com/46277131/77203142-07932300-6af0-11ea-9e18-60913d11b4db.jpg)

Reply:
Oh, sorry, that's right ;-)